WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -130,7 +130,8 @@ private static async Task<AliceClient> RegisterInputAsync(RoundState roundState,
 
 		private async Task ConfirmConnectionAsync(RoundStateUpdater roundStatusUpdater, CancellationToken cancellationToken)
 		{
-			long[] amountsToRequest = { SmartCoin.EffectiveValue(FeeRate, CoordinationFeeRate).Satoshi };
+			var coordinationFeeRate = SmartCoin.Transaction.Transaction.IsLikelyCoinjoin() ? CoordinationFeeRate.Zero : CoordinationFeeRate;
```

Comment:
This includes WW1 CJs, too. Was that your intention?

Reply:
Fixed. The information is coming from the backend with the input registration response.